src/redux/actions/authentication/index.js

```diff
@@ -6,7 +6,7 @@ import loginWithFacebook, {
     AUTH_ERROR_WITH_FACEBOOK,
 } from './facebook-login';
 import loginWithEmailAndPassword, { AUTH_START } from './email-password-login';
-import { AUTH_SUCCESS, AUTH_ERROR } from './handle-email-password-sign-in';
+import { AUTH_SUCCESS, SHOW_ERROR_MESSAGE } from './handle-email-password-sign-in';
```

Comment:
SHOW_ERROR_MESSAGE is meant to be exported only in show-error-message.js and in the app/index.js, not here.